src/flatbuffers/containers/flat_multimap.rs

```diff
@@ -0,0 +1,86 @@
+use std::marker::PhantomData;
+
+use crate::flatbuffers::containers::sorted_index::SortedIndex;
+use flatbuffers::{Follow, Vector};
+
+/// A map-like container that uses flatbuffer references.
+/// Provides O(log n) lookup time using binary search on the sorted index.
+/// I is a key type, Keys is specific container of keys, &[I] for fast indexing (u32, u64)
+/// and flatbuffers::Vector<I> if there is no conversion from Vector (str) to slice.
```

Comment:
Since we're accumulating several of these flatbuffer utility structs, I'm wondering if it makes sense to create and publish a new separate crate just for those? I imagine others in the Rust community could find them useful too.

No need to do so here, but something to consider.

Reply:
That could be an option once we finally stabilize the interface and merge it to master.